clients/gtest/blas3/dgmm_gtest.yaml

```diff
@@ -45,4 +45,33 @@ Tests:
     stride_scale: [ 1.0, 2.5 ]
     api: [ FORTRAN, C ]
     backend_flags: AMD
+
+  - name: dgmm_bad_arg
+    category: pre_checkin
+    function:
+      - dgmm_bad_arg
```

Comment:
can functions `dgmm_batched_bad_arg` and `dgmm_strided_batched_bad_arg` added here. Similarly for all other Level 3 functions as well.